### DISCUSSION TITLE
Muuri Starters May Need A Little Help! :-)

### DISCUSSION THREAD

**Discussion by TiddlyTweeter** [BurningTreeC/tiddlywiki-muuri#6] [Q&A]
I think Muuri is brilliant! ...

BUT the general list does increasingly get queries like this ...

> _Is there someone who would be willing to take me by the hand on how to setup the Muuri plugin?
I installed it and i can now drag tiddlers around but how i can add tiddlers to a grid is completely unclear to me._.. ([link](https://groups.google.com/g/tiddlywiki/c/jKmhBTE4jOk/m/dL5uQeH1AQAJ))_

I actually think here is better than the GG for Muuri. 

Anyway, I just wanted you to know there is much interest in Muuri.
TT

**Comment by rayvermey** [comment]
Thank you!
I already got one answer on how to start but still am not succeeding on setting this up... SORRY ...
Still learning and trying....this is all rather new to me but i learn every day!
Hope someone can make a step by step tutoirial...
After i master the Muuri i would go further with KanBan

Big thanks in advance and please forgive my daftness :-)

Ray

**Comment by TiddlyTweeter** [comment]
Just keep asking. It WILL get easier.

**Comment by BurningTreeC** [comment]
Thank you for your comment @TiddlyTweeter , I have to admit that I lost track on what's going on in the google groups but here I'm still present.

I'm glad that there's much interest in Muuri and I'm open for all questions regarding it

**Reply by TiddlyTweeter** [reply]
Right! Here makes most sense to discuss Muuri. I will on the GG, when I see it relevant, direct posters here.

**Reply by BurningTreeC** [reply]
Thank you @TiddlyTweeter !

**Comment by rayvermey** [comment]
Well still did not get Muuri active on my TW5 file.
Looks like something blocks the settings of  Muuri.
When i download new TW5 and install Muuri it all just works.
Then if i import my own TW5 file it gets messed up again.
I click on the Muuri view and make it say 4 collums, the story view just doesnt change.
The 2 tiddlers i have open don't get tiled so to speak.
Also new tiddlers get in full width beneath the other tiddlers instead of in a smaller column....
WHat is going on??? :-)

Ray
